staging/src/k8s.io/component-base/tracing/utils.go

```diff
@@ -106,3 +108,8 @@ func WrapperFor(tp oteltrace.TracerProvider) transport.WrapperFunc {
 func Propagators() propagation.TextMapPropagator {
 	return propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{})
 }
+
+// Default span name formatter for http req
+func defaultTransportSpanFormatter(_ string, r *http.Request) string {
+	return fmt.Sprintf("%s %s", r.Method, r.URL.Host)
```

Comment:
We should avoid including the full host here.  See https://github.com/open-telemetry/opentelemetry-specification/blob/v1.1.0/specification/trace/semantic_conventions/http.md#name.  It may be better to fix this in upstream otel than to fix it here as well.